app/src/main/java/org/fossasia/susi/ai/adapters/viewholders/ChatViewHolder.java

```diff
@@ -55,4 +70,71 @@ public ChatViewHolder(View view, ClickListener clickListener ,int myMessage) {
             default:
         }
     }
+
+    public void setView(ChatMessage chatMessage, int viewType, int flag, String query, final ChatViewHolder chatViewHolder) {
+        model = chatMessage;
+        if (model != null) {
+            try {
+                switch (viewType) {
+                    case USER_MESSAGE:
+                        chatTextView.setText(model.getContent());
+                        timeStamp.setText(model.getTimeStamp());
+                        if (model.getIsDelivered())
+                            receivedTick.setImageResource(R.drawable.check);
+                        else
+                            receivedTick.setImageResource(R.drawable.clock);
+
+                        chatTextView.setTag(chatViewHolder);
```

Comment:
change this to `chatTextView.setTag(this);` and remove chatViewHolder from parameter.